website/project/model.py

```diff
@@ -2359,15 +2371,19 @@ def add_log(self, action, params, auth, foreign_user=None, log_date=None, save=T
             user=user,
             foreign_user=foreign_user,
             params=params,
+            node=self,
+            original_node=params['node']
```

Comment:
I haven't found any examples of where 'node' or 'project' is not in `params` but I will add the case if neither are present, to default to self.